package.json

```diff
@@ -58,7 +58,7 @@
     "browserama": "^3.1.1",
     "core-js": "^3.7.0",
     "genesys-cloud-client-logger": "^4.0.0",
-    "genesys-cloud-streaming-client": "^13.4.0",
+    "genesys-cloud-streaming-client": "^13.4.1-branch.PCM1773.6",
```

Comment:
what's in this?

Reply:
Just this stuff: https://github.com/purecloudlabs/genesys-cloud-streaming-client/pull/130